parley/src/main/java/nu/parley/android/Parley.java

```diff
@@ -73,6 +73,8 @@ private static class Loader {
     private boolean loadingMore = false;
     private boolean refreshingMessages = false;
 
+    private boolean doRequestPushPermission = true;
```

Comment:
Rename: `requestNotificationPermission`

Reply:
Thanks for your feedback Alex!
I updated the change as suggested.
Could you merge it and make a new version (3.9.2) for us?
Kind regards,
Diederick

Reply:
Thanks! Yep, will do! 👍